### DISCUSSION TITLE
Welcome to Daily-Coding-DS-ALGO-Practice Discussions!

### DISCUSSION THREAD

**Discussion by Ayush7614** [Ayush7614/Daily-Coding-DS-ALGO-Practice#54] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ayush7614** [comment]
Everyone welcome in Daily-DS-ALGO-Practice Projects Anyone can contribute in the project

**Reply by VinayakMohite4040** [reply]
Great Initiative! @Ayush7614

**Reply by Ayush7614** [reply]
> Great Initiative! @Ayush7614

Thanks brother @VinayakMohite4040 you work very good for the project keep contributing

**Comment by payaldutta** [comment]
Umm Actually, I am totally new to open source so can someone help me in getting started.

**Reply by akshatnema** [reply]
I think , to contribute to this repository, you should first well versed about Git and GitHub. You can better watch Git tutorials on YouTube and then can contribute to this in your own way.
